modules/kernel/src/main/java/org/opencastproject/kernel/rest/RestPublisher.java

```diff
@@ -452,6 +450,8 @@ public URL getResource(String name) {
    */
   class StaticResourceBundleTracker extends BundleTracker<Object> {
 
+    private HashMap<Bundle, ServiceRegistration> servlets = new HashMap<>();
```

Comment:
What about thread-safety when accessing this data structure?

Reply:
fixed